Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM php:8.1-apache
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+      procps \
+      nano \
+      git \
+      unzip \
+      libicu-dev \
+      zlib1g-dev \
+      libxml2 \
+      libxml2-dev \
+      libreadline-dev \
+      supervisor \
+      cron \
+      sudo \
+      libzip-dev \
+      wget \
+      librabbitmq-dev \
```

Comment:
This Dockerfile looks like a copy paste from another project, but it's ok. We can leave it like this and improve it later.

Reply:
That's right. I needed something for fast setting up a project without installing apache on my own at my MacBook.

I will keep it as it is and I would be improving it during next iterations.